ui/schema/index.js

```diff
@@ -36,6 +36,22 @@ function TruncateLink({ link, email }) {
 
 const sentenceCase = (str) => upperFirst(str.replaceAll('-', ' '));
 
+const DocumentationLink = ({ link, title }) => (
+  <>
+    <a
+      href={link}
+      rel="noreferrer"
+      target="_blank"
```

Comment:
nextjs Link component does not accept a "title" attribute, so I'm just using vanilla links

Reply:
FWIW, this isn't using Next.js `<Link>` components, it's using https://github.com/nhsx/standards-registry/blob/master/ui/components/Link/index.js so would have supported a `title` attribute (with a small amount of work), but it's merged now, so whatevs.

Also Next.js `<Link>`s _kind of_ support `title`, you just need to pass the `<a>` as a child.

e.g.

```
<Link href="/">
  <a title="Home">Home</a>
</Link>
```